### PR TITLE
new code

### PR DESCRIPTION
**Pull request by jhdaguitera**
I wanted to upload what I have so far. I tried to be descriptive in the comments to explain. I have a bunch of print statements to help me see what's going on or what is supposed to be happening theoretically.

I am still trying to figure out how to portion out percentages of cash to trade each time. 

Hopefully no conflicts this time around

### CODE REVIEW

File: ROC_RSI/main.py [hunk 1]
```diff
@@ -7,34 +7,71 @@
 
 
 class SMA_ROC_RSI(QCAlgorithm):  
-    def Initialize(self):
+    def __init__(self):
```
Comment (scheffeltravis) [inline ROC_RSI/main.py:10]: We were trying to standardize on using the `Initialize` class method so as to not run into problems with different versions of python.

You should be able to simply throw all of this under that method again without an issue.

**Review by scheffeltravis — COMMENTED** [review]
If you're looking for a way to use a portion of your available liquid cash for trading from your portfolio, [this code snippet](https://github.com/scheffeltravis/Stock-Trading-Algorithms/blob/71e6b5f090a30b8af41743b7a5fca34433d5363f/climax/main.py#L45-L47) from my first climax algo iteration might be helpful. It simply takes 10% of my available cash and submits a limit order for the amount of shares I can purchase with it.

**Review by silardi — APPROVED** [review]
As Travis said just move the code from `__init__` back to `Initialize` and remove `__init__`. Everything else looks good to me.

**Comment by jhdaguitera** [comment]
thanks - sorry I had'nt worked on this for a few days...needed a break. Good catch - looks like didn't revise the _init_ thing after I recopy/pasted from that SMA algo. I will make the fix. @scheffeltravis thanks for the code snippet, I will try to make use of it

**Pull request closed without merging**